pypit/requirements.txt

```diff
@@ -7,7 +7,7 @@
 numpy>=1.11.2
 scipy>=0.18.1
 matplotlib>=1.5.3
-astropy>=1.2.1
+astropy>=1.3.0
```

Comment:
we should probably raise this up to 2.0 or higher

Reply:
pushed it to 2.0.5, but we could set it to >3 something.